test.rb

```diff
@@ -0,0 +1,3 @@
+puts 'Hello World'
+puts "hollow world"
```

Comment:
C: Prefer single-quoted strings when you don't need string interpolation or special symbols.